content/v2.0-preview/_index.md

```diff
@@ -1,51 +1,53 @@
 ---
-title: "Overview"
+title: "Welcome"
 weight: -1
 cascade:
     version: "2.0-preview"
 ---
 
-{{< img src="/media/banner.png" alt="Crossplane Popsicle Truck" size="large" >}}
+Welcome to the Crossplane documentation. Crossplane is a control plane framework
+for platform engineering. 
 
-<br />
+Crossplane lets you build control planes to manage your cloud native software.
+It lets you design the APIs and abstractions that your users use to interact
+with your control planes.
 
-Crossplane is an open source Kubernetes extension that transforms your Kubernetes 
-cluster into a **universal control plane**. 
+Crossplane has a rich ecosystem of extensions that make building a control plane
+faster and easier. It's built on Kubernetes, so it works with all the Kubernetes
+tools you already use.
 
-Crossplane lets you manage anything, anywhere, all through standard Kubernetes
-APIs. Crossplane can even let you 
-[order a pizza](https://blog.crossplane.io/providers-101-ordering-pizza-with-kubernetes-and-crossplane/) 
-directly from Kubernetes. If it has an API, Crossplane can connect to it. 
+{{< hint "tip" >}}
+**A control plane is software that controls other software.**
```

Comment:
i like ana's idea of immediately giving the practical example of `The major cloud providers are all built using control planes` right here near the top instead of the very end.

Reply:
Okay, I'll do that.